index.html

```diff
@@ -90,10 +92,10 @@
 })(window);
 
 (function(self) {
-  var entityMap = { "&": "&amp;", "<": "&lt;", ">": "&gt;", '"': '&quot;', "'": '&#39;', "/": '&#x2F;' };
+  var entityMap = { "&": "&amp;", "<": "&lt;", ">": "&gt;", '"': '&quot;', "'": '&#39;', "/": '&#x2F;' , "\n": '<br>' };
```

Comment:
This function is used in several places - in particular, attribute values on html elements - where the `<br>` tag is not acceptable.

Either the `\n` to `<br>` transformation should be a separate function, or it should be optional (selected based on a function parameter, perhaps).

Reply:
Instead of doing `\n` to `<br>` escaping here, consider setting [`style="white-space: pre-wrap"`](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space#pre-wrap) on the edit note preview.